StringCalculator.tests/StringCalculatorTests.cs

```diff
@@ -0,0 +1,111 @@
+﻿using NUnit.Framework;
+
+namespace StringCalculator.tests;
+public class StringCalculatorTests
+{
+    private StringCalculator _calculator = null!;
+
+    [SetUp]
+    public void Setup()
+    {
+        _calculator = new StringCalculator();
+    }
+
+    [TestCase ("")]
```

Comment:
I like that you found the TestCase and are using it, but let's sit to having the input value in the method itself for now so you can better understand and practice the concepts around arrange, act, assert.